app/services/fetchFromApi.server.ts

```diff
@@ -0,0 +1,35 @@
+type FetchFromApiOptions = {
+  url: string;
+  errorMessage: string;
+};
+
+const userId = "PierreM";
```

Comment:
NIt: Maybe we add a comment here for later...
```suggestion
// this is a fixed identifier, which will later be replaced by a suitable authentication.
const userId = "PierreM";
```